pom.xml

```diff
@@ -569,6 +569,56 @@
                 <jni.classifier>${cuda.version}-arm64</jni.classifier>
             </properties>
         </profile>
+        <profile>
+            <id>bloopInstall</id>
+            <activation>
```

Comment:
I agree with the sentiment of the question. Initially had it without it but then added it for consistency because we have it almost everywhere. We can discuss on the actual PR to NVIDIA/spark-rapids